app/views/payment_tracking/index.html.erb

```diff
@@ -48,6 +49,7 @@
           <td><%= if rental.renter.department then rental.renter.department.name else rental.renter.full_name end %></td>
           <td><%= ((Time.current-rental.created_at) / 1.days ).floor %></td>
           <td><%= number_to_currency rental.balance %></td>
+          <td><%= rental.payments.map { |ft| ft.transactable.reference }.join ', ' %></td>
```

Comment:
Is it worth checking for payments.empty? beforehand? Stylistic choice...

rent.pay.map {} unless pay.empty?